src/datapoint.py

```diff
@@ -0,0 +1,114 @@
+from typing import Dict
+import numpy as np
+from enum import Enum
+from abc import ABC
+
+
+class Method(Enum):
+    C = 'Classification'
+    R = 'Regression'
+
+
+class DataSet:
+
+    def __init__(self, data: Dict[str, np.ndarray], feature_predict_name: str, method: Method):
+        """
+        :param: data the data which will be the input to the decision tree
+        :param: Refers to the method used 'R' is for regression or 'C' is for classification
+        """
```

Comment:
Change ces docstring a un format Google
```suggestion
        """Courte description.
        
        Longue description.
        
        Args:
            data: 
            feature_predict_name: 
            
        Returns:
            description de ce que tu renvoie magueule
        """
```